plugins/armorblox/icon_armorblox/triggers/get_incidents/schema.py

```diff
@@ -0,0 +1,329 @@
+# GENERATED BY KOMAND SDK - DO NOT EDIT
+import insightconnect_plugin_runtime
+import json
+
+
+class Component:
+    DESCRIPTION = "Get a list of incidents identified by Armorblox, by default it starts polling all the incidents since last day"
+
+
+class Input:
+    
+    INTERVAL = "interval"
+    
+
+class Output:
+    
+    INCIDENTS = "incidents"
+    
+
+class GetIncidentsInput(insightconnect_plugin_runtime.Input):
+    schema = json.loads("""
+   {
+  "type": "object",
+  "title": "Variables",
+  "properties": {
+    "interval": {
+      "type": "integer",
+      "title": "Fetch_Interval",
+      "description": "Polling inteval in seconds",
+      "default": 600,
+      "order": 1
+    }
+  }
+}
+    """)
+
+    def __init__(self):
+        super(self.__class__, self).__init__(self.schema)
+
+
+class GetIncidentsOutput(insightconnect_plugin_runtime.Output):
+    schema = json.loads("""
+   {
+  "type": "object",
+  "title": "Variables",
+  "properties": {
+    "incidents": {
+      "type": "array",
+      "title": "Incidents",
+      "description": "A list of 100 incidents identified by Armorblox",
```

Comment:
I don't think we should say 100 here - not sure how pagination is controlled

Reply:
yes @urajat , I think "A list of incidents identified by Armorblox" - this description should be okay..